controls/SV-257850.rb

```diff
@@ -25,4 +25,37 @@
   tag 'documentable'
   tag cci: ['CCI-001764']
   tag nist: ['CM-7 (2)']
+  tag 'host'
+
+  only_if('Control not applicable within a container', impact: 0.0) {
+    !virtualization.system.eql?('docker')
+  }
+
+  interactive_users = passwd.where {
+    uid.to_i >= 1000 && shell !~ /nologin/
```

Comment:
here we are just checking that our shell does not match nologin, but SV-257843 checks against a list of non-interactive shells.  do we not need to check against that full list here?